src/components/aktivitet-filterform/aktivitet-filterform.tsx

```diff
@@ -1,24 +1,23 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
 import SubmitKnapp from './../submit-knapp';
-import { AktiviteterValg, FiltreringAktiviteterValg } from '../../ducks/filtrering';
+import {FiltreringAktiviteterValg} from '../../ducks/filtrering';
 import './aktivitet-filterform.less';
 
 const aktivitetInitialState: FiltreringAktiviteterValg = {
-    'BEHANDLING': AktiviteterValg.NA,
-    'EGEN': AktiviteterValg.NA,
-    'GRUPPEAKTIVITET': AktiviteterValg.NA,
-    'IJOBB': AktiviteterValg.NA,
-    'MOTE': AktiviteterValg.NA,
-    'SOKEAVTALE': AktiviteterValg.NA,
-    'STILLING': AktiviteterValg.NA,
-    'TILTAK': AktiviteterValg.NA,
-    'UTDANNINGAKTIVITET': AktiviteterValg.NA
+    'BEHANDLING': null,
```

Comment:
detta brekker deserialisering på backend https://github.com/navikt/veilarbportefolje/blob/b8522e42f9af61076bc3eeece394701811e00453/src/main/java/no/nav/pto/veilarbportefolje/feed/aktivitet/AktivitetFiltervalg.java#L3

Reply:
@alexandraback is it possible to change backend? Is there any special reason why NA is used on backend?

Reply:
no idea why it is NA. What is the benefit of changing it to null?

Reply:
we had trouble to investigate if object is empty....we checked for null values, empty string, empty arrays...and then we encountered those NA values..

Reply:
rollback to original version, it should be better now